jre_emul/build_subset_frameworks.sh

```diff
@@ -17,7 +17,7 @@
 # 1. cd to the j2objc/jre_emul directory
 # 2. run "./build_subset_frameworks.sh"
 #
-# To delete these frameworks, run "./build_subset_frameworks clean".
+# To delete these frameworks, run "./build_subset_frameworks.sh clean".
 
 for mk in \
   android_util_framework.mk \
```

Comment:
Please remove the android_util_framework.mk line, and delete that file. The android_util library isn't built any, and there's an error reported when this script runs this make command.

Reply:
Done.